types/src/grantRounds.d.ts

```diff
@@ -25,6 +25,11 @@ export type GrantRoundMetadata = {
   name: string;
   description: string;
   grants: BigNumberish[];
+  properties: {
+    projectWebsite: string;
+    projectGithub: string;
+    twitterHandle: string;
```

Comment:
Our naming convention feels misleading IMO, e.g. using a name "_project_ Website" for the _round_ metadata is confusing. I think we should either:
1. Use more generic names that don't include the words "round" or "project". For example `websiteURI` and `twitterURI` would be the field names for both GrantRoundMetadata and GrantMetadata. This is my preference, since it's also consistent with `logoURI`. I mentioned this [here](https://github.com/dcgtc/dgrants/pull/158#discussion_r698613804), so we'll need to coordinate with @gdixon if we change the names
2. Rename these to reflect they are for rounds, not projects, e.g. `roundWebsite` instead of `projectWebsite`

Reply:
I lean more toward the first option where we have the elements (websiteURI, twitterURI) under each type (GrantRoundMetadata, GrantMetadata) have consistent names. We already have the context from the object that owns the element.